qiskit_experiments/calibration_management/base_calibration_experiment.py

```diff
@@ -298,64 +218,6 @@ def _validate_parameters(self, schedule: ScheduleBlock, n_expected_parameters: i
                 f"{n_expected_parameters} parameters. Found {len(schedule.parameters)}."
             )
 
-    def get_schedule(
-        self,
-        qubits: Optional[Tuple[int, ...]] = None,
-        sched_name: Optional[str] = None,
-        option_name: str = "schedule",
-        assign_params: Optional[Dict[str, Parameter]] = None,
-        **kwargs,
-    ) -> ScheduleBlock:
-        """Get the schedules for the circuits.
-
-        This method defines the order in which the schedules are consumed. This order is
-
-        #. Use the schedules directly available in the experiment, i.e. those specified
-           by experiment users. This is made possible in experiments by implementing the
-           :meth:`get_schedules_from_options` method.
-
-        #. Use the schedules found in the instance of :class:`Calibrations` attached to the
-           experiment. This is done by implementing the :meth:`get_schedules_from_calibrations`
-           method.
-
-        #. Use any default schedules specified by the :meth:`get_schedules_from_defaults`.
-
-        If any one step does not return a schedule then we attempt to get schedules from the next
-        step. If none of these three steps have returned any schedules then an error is raised.
-
-        Args:
-            qubits: The qubits for which to get the schedule in the calibrations. If None is given
-                this will default to the physical qubits of the experiment.
-            sched_name: The name of the schedule to retrieve from the instance of
-                :class:`Calibrations` stored as a protected variable. If this is None then
-                :meth:`get_schedule_from_calibrations` will default to the :code:`self._sched_name`.
-            option_name: The name of the option under which to get the schedule from the experiment
-                options. This will default to "schedule" if None is given.
-            assign_params: A dict that :meth:`get_schedule_from_calibrations` can use to leave
-                certain parameters in the schedule unassigned. The key is the name of the parameter
-                and the value should be an instance of :class:`ParameterExpression`.
-            kwargs: Additional keyword arguments that can be used by implementations of
-                :meth:`get_schedule_from_defaults`.
-
-        Returns:
-            schedules: The schedules (possibly with one or more free parameters) as either a
-                ScheduleBlock or a list of ScheduleBlocks depending on the experiment.
-
-        Raises:
-            CalibrationError: if none of the methods above returned schedules.
-        """
-        schedules = self._get_schedule_from_options(option_name)
-
-        if schedules is None:
-            schedules = self._get_schedule_from_calibrations(qubits, sched_name, assign_params)
-
-        if schedules is None:
-            schedules = self._get_schedule_from_defaults(**kwargs)
-
-        self._validate_schedule(schedules)
```

Comment:
You might as well remove `_validate_schedules` as well since it is now not called by the base class or any subclasses.

Reply:
I removed it, see here: 4db1ab3